package.json

```diff
@@ -24,19 +24,19 @@
   ],
   "scripts": {
     "start": "webpack-dev-server --config webpack/dev.js",
-    "build": "npm run clean && npm run build:lib && npm run build:es6 && mkdir umd && npm run build:umd",
+    "clean": "rimraf lib es6 umd",
+    "build": "npm-run-all -s clean -p build:** -s compress:umd",
```

Comment:
instead of `&&` and `&` using `npm-run-all` to manage dependent tasks.

`compress:umd` will be executed only when all the builds are over.